tests/YouTrackSharp.Tests/Integration/AgileBoards/CreateAgileBoard.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Net.Http;
+using System.Threading.Tasks;
+using Xunit;
+using YouTrackSharp.AgileBoards;
+using YouTrackSharp.Tests.Infrastructure;
+
+namespace YouTrackSharp.Tests.Integration.AgileBoards
+{
+    public partial class AgileBoardServiceTests
+    {
+        public class CreateAgileBoard
+        {
+            /*
```

Comment:
Can you make the tests `[Fact(Ignore="...")]` with that info? Then we have the test but it will be skipped.